lib/ansible/modules/notification/teams.py

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2017, Aymeric Bringard <diadzine@gmail.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+ANSIBLE_METADATA = {'status': ['preview'],
+                    'supported_by': 'community',
+                    'version': '1.0'}
+
+DOCUMENTATION = """
+module: teams
+short_description: Send Microsoft Teams notifications
+description:
+    - The C(teams) module sends notifications to
+      U(https://teams.microsoft.com/start) through Office 365 Connectors via
+      the Incoming WebHook integration.
+version_added: "2.3"
+author: "Aymeric Bringard (@diadzine)"
+options:
+  title:
+    description:
+      - A message can also contain a title.
+    required: false
+    default: None
```

Comment:
If there's no default, `None' is implicit. (It means parameter undefined)
So this is not needed.